src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { JSXElementConstructor } from 'react';
+
+import styles from './IconButton.module.scss';
+import { IconProps } from '../Icons/Icons.types';
+
+export interface Properties {
+  className?: string;
+  onClick: () => void;
+
+  Icon: JSXElementConstructor<IconProps>;
+  label?: string;
+  size?: string | number;
```

Comment:
What are the valid `string`s for `size` here?

Reply:
Same as an [Icon](https://github.com/zer0-os/zUI/blob/master/src/components/Icons/Icons.types.ts#L1).